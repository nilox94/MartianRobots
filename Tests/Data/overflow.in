2 2
0 0 E
FFFF
2 0 N
FFFF
2 2 W
FFFF
0 2 S
FFFF
