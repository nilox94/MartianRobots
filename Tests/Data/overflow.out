2 0 E LOST
2 2 N LOST
0 2 W LOST
0 0 S LOST
